pyproject.toml

```diff
@@ -85,3 +89,14 @@ show_contexts = true
 [tool.mypy]
 mypy_path = "$MYPY_CONFIG_FILE_DIR/ebcl"
 explicit_package_bases = true
+
+[tool.flake8]
+ignore = ['C901']
+per-file-ignores = [
+    'ebcl/common/dependency.py:F841',
+    'ebcl/tools/proxy/proxy.py:F841',
+]
+count = true
+statistics = true
+max-line-length = 120
```

Comment:
127 or 120?

Reply:
120. 127 was the default from the GH action template.